### DISCUSSION TITLE
DeployResources & DeployFunctions Jobs Skipped

### DISCUSSION THREAD

**Discussion by DG-RMD** [microsoftgraph/group-membership-management#17] [Q&A]
Following the example instructions verbatim, when running the Pipeline, the last 2 jobs for DeployResources & DeployFunctions are being skipped and I'm unable to determine why
![gmm](https://user-images.githubusercontent.com/98605425/152059152-900da576-59e6-4f2e-b850-dfc2e1d40e76.PNG)

Any ideas as to what step I'm missing/where I should be looking?

**Comment by alrios-ms** [comment]
Hi @DG-RMD 

The conditions defined in [vsts-cicd.yml](https://github.com/microsoftgraph/group-membership-management/blob/55ecdba6bd2e3fc2ea9dea2831311cd7dc0caa3f/vsts-cicd.yml) will determine whether a particular stage should run or not, verify that your branch name matches the value defined in your vsts-cicd.yml. For instance, here the branch must be master.

https://github.com/microsoftgraph/group-membership-management/blob/55ecdba6bd2e3fc2ea9dea2831311cd7dc0caa3f/vsts-cicd.yml#L121-L127
